framework/src/application/network/network.ts

```diff
@@ -158,27 +183,28 @@ export class Network {
 		const fixedPeers = this._options.fixedPeers
 			? this._options.fixedPeers.map(peer => ({
 					ipAddress: peer.ip,
-					wsPort: peer.wsPort,
+					port: peer.wsPort,
 			  }))
 			: [];
 
 		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
 		const whitelistedPeers = this._options.whitelistedPeers
 			? this._options.whitelistedPeers.map(peer => ({
 					ipAddress: peer.ip,
-					wsPort: peer.wsPort,
+					port: peer.wsPort,
 			  }))
 			: [];
 
 		const p2pConfig: liskP2P.p2pTypes.P2PConfig = {
+			port: this._options.wsPort,
 			nodeInfo: initialNodeInfo,
 			hostIp: this._options.hostIp,
 			blacklistedIPs,
 			fixedPeers,
 			whitelistedPeers,
 			seedPeers: seedPeers.map(peer => ({
 				ipAddress: peer.ip,
-				wsPort: peer.wsPort,
+				port: peer.wsPort,
```

Comment:
Lets add another issue to change wsPort => port in config

Reply:
created https://github.com/LiskHQ/lisk-sdk/issues/5527